test/services/MeasureService.test.ts

```diff
@@ -345,6 +346,74 @@ describe('MeasureService', () => {
         });
     });
   });
+
+  describe('$data-requirements', () => {
+    // spy on calculation function
+    jest.spyOn(Calculator, 'calculateDataRequirements').mockResolvedValue({
+      results: {
+        resourceType: 'Library',
+        type: {
+          coding: [{ code: 'module-definition', system: 'http://terminology.hl7.org/CodeSystem/library-type' }]
+        },
+        status: 'draft',
+        dataRequirement: []
+      }
+    });
+
+    it('returns 200 and a Library for a simple measure with dependencies and no period params', async () => {
+      await supertest(server.app)
+        .post('/4_0_1/Measure/$data-requirements')
+        .send({ resourceType: 'Parameters', parameter: [{ name: 'id', valueString: 'testWithRootLibAndDeps' }] })
+        .set('content-type', 'application/fhir+json')
+        .expect(200)
+        .then(response => {
```

Comment:
According to the spec: "The result of the requirements operation is a _module-definition_ Library that returns the computed effective requirements of the artifact." Do you think it's worth adding a check in these unit tests that the library-type of the returned FHIR Library is "module-definition"?

Reply:
I can add this, but looking at it, it's really just checking the values on the mocked return item just above, created by the spy. Maybe it's still good to have from future-proof perspective if we ever go in a different testing direction.